service/history/replication/stream_sender.go

```diff
@@ -403,3 +549,12 @@ func (s *StreamSenderImpl) sendToStream(payload *historyservice.StreamWorkflowRe
 	}
 	return nil
 }
+
+func (s *StreamSenderImpl) getTaskPriority(task tasks.Task) enumsspb.TaskPriority {
+	switch t := task.(type) {
+	case *tasks.SyncWorkflowStateTask:
+		return t.Priority
```

Comment:
This is a good point. The change that assign Priority to `SyncWorkflowState` task is already merged [here](https://github.com/temporalio/temporal/blob/main/service/history/workflow/task_generator.go#L765), and will be at least 2 version ahead of enabling this change.

I will change this method to avoid returning TASK_PRIORITY_UNSPECIFIED to be safer.

Reply:
Method updated.